superset/assets/src/explore/components/ControlPanelSection.jsx

```diff
@@ -31,21 +31,21 @@ export default class ControlPanelSection extends React.Component {
     return (
       label &&
         <div>
+          <span>
+            <span onClick={this.toggleExpand.bind(this)}>{label}</span>
+            {' '}
```

Comment:
do we need this still?

Reply:
it's more like a `&nbsp;` but looks a bit better